lstchain/reco/r0_to_dl1.py

```diff
@@ -451,9 +451,17 @@ def r0_to_dl1(
                         'HillasParameterizationError in get_dl1()'
                     )
 
-                # The condition below should now be true for all events, this
-                # is a relic of previous approach in which only survivors of
-                # cleaning and parametrization were further processed.
+                # Workaround, see below.
+                if i == 0 and dl1_container.n_pixels == 0:
```

Comment:
what if the second event `(i == 1)` also has `dl1_container.n_pixels == 0`?

Reply:
Could the condition be extended to the first `n` events?

Reply:
Could happen I guess, but only if normal triggers are not activated and calib fox is off. Will fix this and also display a message when an event is skipped.

Reply:
I am working on a cleaner fix, I will put this PR back to draft.

Reply:
I reverted the workaround to solve #518, will be done in a separate PR

Reply:
so this error does not show up anymore when #523 is merged? I guess @moralejo you tested with one of those files found in #518, right?

Reply:
I tested it with one of the failing subruns, and with the previous (normal) subrun, and it works in both cases. Will ask @morcuended  to try it on a full night.